testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -524,6 +537,17 @@ QUnit.module('interaction via swipe', {
         assert.strictEqual(startEvent.maxRightOffset, 0, 'container was not be moved');
     });
 
+    QUnit.test('item container should not be moved right if first item is disabled and next is selected', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [{ disabled: true }, 2, 3],
+            selectedIndex: 1,
+        });
+
+        const startEvent = pointerMock($multiView).start().swipeStart().lastEvent();
+
+        assert.strictEqual(startEvent.maxRightOffset, 0, 'container was not be moved');
```

Comment:
Let's fix in all your tests
```suggestion
        assert.strictEqual(startEvent.maxRightOffset, 0, 'container was not moved');
```

Reply:
Fixed here: [1254ebd2f865395198a8f28647dccaf61766bada].